wire-library/wire-tests-swift/src/main/swift/proto3/AllTypes.swift

```diff
@@ -0,0 +1,2189 @@
+        }
+        set {
+            copyStorage()
+            storage.sfixed32 = newValue
+        }
+    }
+    public var int64: Int64 {
+        get {
+            storage.int64
+        }
+        set {
+            copyStorage()
+            storage.int64 = newValue
+        }
+    }
+    public var uint64: UInt64 {
+        get {
+            storage.uint64
+        }
+        set {
+            copyStorage()
+            storage.uint64 = newValue
+        }
+    }
+    public var sint64: Int64 {
+        get {
+            storage.sint64
+        }
+        set {
+            copyStorage()
+            storage.sint64 = newValue
+        }
+    }
+    public var fixed64: UInt64 {
+        get {
+            storage.fixed64
+        }
+        set {
+            copyStorage()
+            storage.fixed64 = newValue
+        }
+    }
+    public var sfixed64: Int64 {
+        get {
+            storage.sfixed64
+        }
+        set {
+            copyStorage()
+            storage.sfixed64 = newValue
+        }
+    }
+    public var bool: Bool {
+        get {
+            storage.bool
+        }
+        set {
+            copyStorage()
+            storage.bool = newValue
+        }
+    }
+    public var float: Float {
+        get {
+            storage.float
+        }
+        set {
+            copyStorage()
+            storage.float = newValue
+        }
+    }
+    public var double: Double {
+        get {
+            storage.double
+        }
+        set {
+            copyStorage()
+            storage.double = newValue
+        }
+    }
+    public var string: String {
+        get {
+            storage.string
+        }
+        set {
+            copyStorage()
+            storage.string = newValue
+        }
+    }
+    public var bytes: Data {
+        get {
+            storage.bytes
+        }
+        set {
+            copyStorage()
+            storage.bytes = newValue
+        }
+    }
+    public var nested_enum: NestedEnum {
+        get {
+            storage.nested_enum
+        }
+        set {
+            copyStorage()
+            storage.nested_enum = newValue
+        }
+    }
+    public var nested_message: NestedMessage? {
+        get {
+            storage.nested_message
+        }
+        set {
+            copyStorage()
+            storage.nested_message = newValue
+        }
+    }
+    public var any: Any? {
+        get {
+            storage.any
+        }
+        set {
+            copyStorage()
+            storage.any = newValue
+        }
+    }
+    public var duration: Duration? {
+        get {
+            storage.duration
+        }
+        set {
+            copyStorage()
+            storage.duration = newValue
+        }
+    }
+    public var `struct`: Struct? {
+        get {
+            storage.`struct`
+        }
+        set {
+            copyStorage()
+            storage.`struct` = newValue
+        }
+    }
+    public var list_value: ListValue? {
```

Comment:
This won't compile right now, right? Since we have things like `ListValue` and `Struct` in here? We would need to be importing the basic Google types for that to work (for now), but eventually we want to either be using all-included types, or native Swift types, with no dependency on the actual Google library.

So, options seem like:
1. It won't build, but we don't care for now and our goal is to get it building
2. Immediately add all of these Google types into Wire, like we did with Duration, then work to remove ones that can be modeled with native types instead
3. Set up the test to depend on the actual Google basic types protos (generated as a separate module) and slowly work to remove that dependency

Any of these seem fine to me, although if it's currently building/passing I'd hate to turn off all other tests in order to go with (1). If we can still run other tests for now then all seem like fine options. I'd lean mildly toward (1) or (3) just so that we're not technically temporarily adding API with (2) (like the Struct type), then removing it as we move toward using native Swift types.

Reply:
Or we could: 

4. Generate a "golden" proto -- one for each WellKnownType we implement. Once we have every WellKnownType included, remove the fragmented golden protos in favor of `all_types.proto`.

Reply:
Also good with that, assuming other maintainers are cool with adding/removing golden protos. Could also make them iOS-specific golden protos in some new directory somewhere.

I leave it up to you.